tutorials/scripting/c_sharp/c_sharp_features.rst

```diff
@@ -81,8 +81,19 @@ otherwise it returns true. Note that using the ``is`` operator against ``null``
         // This block can never happen.
     }
 
-For more advanced type checking, you can look into `Pattern Matching <https://docs.microsoft.com/en-us/dotnet/csharp/pattern-matching>`_.
+**Type test with variable declaration**
```

Comment:
I'm not sure this needs to be a new section, seems like it could be part of the previous one.

Reply:
Hmm. Yeah, reworded and put the "advanced" note back where it was.